scripts/devcontainer/_wait-until-apiserver-ready

```diff
@@ -13,7 +13,6 @@ function wait_for {
   done
 }
 
-wait_for analysis.js
```

Comment:
This should be replaced with a blazor file

Reply:
good catch; done